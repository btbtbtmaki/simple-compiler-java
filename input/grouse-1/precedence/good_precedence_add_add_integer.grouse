main {

  print 47 + 55 + 88 nl;
  print 42 + 62 + 77 nl;
  print 50 + 68 + 29 nl;
  print 47 + 59 + 40 nl;
  print 88 + 72 + 49 nl;
  print 53 + 73 + 46 nl;
  print 6 + 69 + 82 nl;
  print 55 + 87 + 50 nl;
  print 84 + 10 + 67 nl;
  print 14 + 56 + 39 nl;
  print 55 + 85 + 17 nl;
  print 17 + 27 + 54 nl;

  print 99 + 90 + 17 nl;
  print 53 + 38 + 1 nl;
  print 23 + 21 + 58 nl;
  print 91 + 34 + 73 nl;
  print 6 + 63 + 40 nl;
  print 51 + 53 + 96 nl;
  print 92 + 6 + 98 nl;
  print 16 + 14 + 92 nl;
  print 97 + 31 + 46 nl;
  print 57 + 60 + 67 nl;
  print 47 + 29 + 34 nl;
  print 59 + 32 + 74 nl;

  print 32 + 77 + 75 + 19 nl;
  print 77 + 77 + 69 + 58 nl;
  print 44 + 69 + 19 + 6 nl;
  print 1 + 47 + 59 + 9 nl;
  print 52 + 74 + 45 + 71 nl;
  print 20 + 83 + 18 + 58 nl;
  print 51 + 56 + 38 + 99 nl;
  print 49 + 36 + 4 + 13 nl;
  print 24 + 54 + 88 + 86 nl;
  print 95 + 16 + 49 + 88 nl;
  print 65 + 32 + 59 + 86 nl;
  print 55 + 4 + 38 + 20 nl;

  print 90 + 10 + 94 + 45 nl;
  print 69 + 38 + 98 + 93 nl;
  print 20 + 44 + 51 + 43 nl;
  print 92 + 19 + 16 + 1 nl;
  print 52 + 4 + 97 + 41 nl;
  print 94 + 32 + 57 + 41 nl;
  print 51 + 10 + 50 + 5 nl;
  print 16 + 39 + 46 + 95 nl;
  print 87 + 68 + 68 + 82 nl;
  print 95 + 61 + 33 + 91 nl;
  print 97 + 18 + 89 + 81 nl;
  print 33 + 41 + 94 + 87 nl;

}

