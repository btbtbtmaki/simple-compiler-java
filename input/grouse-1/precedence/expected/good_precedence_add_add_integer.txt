190
181
147
146
209
172
157
192
161
109
157
98
206
92
102
198
109
200
196
122
174
184
110
165
203
281
138
116
242
179
244
102
252
248
242
117
239
298
158
128
194
224
116
196
305
280
285
255
